com.unity.perception/Documentation~/Ground Truth/KeypointLabeler.md

```diff
@@ -0,0 +1,88 @@
+# Keypoint Labeler
```

Comment:
I'm wondering if we should add a link to the keypoint tutorial somewhere here in this doc

Reply:
Good idea. Added to Setup section.